addons/ar/arca/bill.go

```diff
@@ -0,0 +1,255 @@
+package arca
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/l10n"
+	"github.com/invopop/gobl/org"
+	"github.com/invopop/gobl/pay"
+	"github.com/invopop/gobl/tax"
+	"github.com/invopop/validation"
+)
+
+var invoiceCorrectionDefinitions = tax.CorrectionSet{
+	{
+		Schema: bill.ShortSchemaInvoice,
+		Types: []cbc.Key{
+			bill.InvoiceTypeCreditNote,
+			bill.InvoiceTypeDebitNote,
+		},
+		Extensions: []cbc.Key{
+			ExtKeyDocType,
+		},
+	},
+}
+
+func normalizeInvoice(inv *bill.Invoice) {
+	if inv.Customer != nil && !inv.Customer.Ext.Has(ExtKeyVATStatus) {
+		switch {
+		case inv.Customer.TaxID == nil:
+			inv.Customer.Ext = inv.Customer.Ext.Set(ExtKeyVATStatus, "5") // Final Consumer
+		case inv.Customer.TaxID.Country == l10n.AR.Tax():
+			inv.Customer.Ext = inv.Customer.Ext.Set(ExtKeyVATStatus, "1") // Registered VAT Company
+		default:
+			inv.Customer.Ext = inv.Customer.Ext.Set(ExtKeyVATStatus, "9") // Foreign Customer
+		}
+	}
```

Comment:
I'd suggest putting this inside its own method with the party for greater clarity and to avoid calling `inv.Customer` all the time.

The other issue here as this is not deterministic. If the VATStatus is already set, the meaning may be mis-matched. Is there a reason why it cannot be forced?